src/Core/Framework/Adapter/Cache/CacheClearer.php

```diff
@@ -84,13 +85,16 @@ public function clearContainerCache(): void
         }
 
         $finder = (new Finder())->in($this->cacheDir)->name('*Container*')->depth(0);
-        $containerCaches = [];
 
         foreach ($finder->getIterator() as $containerPaths) {
-            $containerCaches[] = $containerPaths->getRealPath();
+            try {
+                $this->filesystem->remove($containerPaths->getRealPath());
+            } catch (IOException $e) {
+                // If the path is not readable, we skip it
```

Comment:
can we make it more precise that it can happen when another process deletes the files concurrently?